src/pages/components/Input/Input.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { forwardRef, InputHTMLAttributes } from 'react';
+
+interface InputPropsInterface extends InputHTMLAttributes<HTMLInputElement> {
+  header: string;
+  type: string;
+}
+
+type Ref = HTMLInputElement;
```

Comment:
is it possible to use interface?

Reply:
In this case, it is best to use `type`.

Reply:
why? it seems like you don't need to create the `Ref` type and you can directly use the `HTMLInputElement` interface